### DISCUSSION TITLE
SEO data for collection

### DISCUSSION THREAD

**Discussion by solidevolution** [Shopify/storefront-api-feedback#70] [Ideas & Requests]
Hello,

currently it is possible to save collection SEO fields in Shopify but you don't get them via the Storefront API:

https://shopify.dev/api/storefront/2022-01/objects/Collection#top
https://shopify.dev/api/storefront/2022-01/objects/SEO#top

**Comment by corygibbons** [comment]
Looks like this is added in [`2022-04`](https://shopify.dev/api/storefront/2022-04/objects/Collection#field-collection-seo) 👍
